goal_src/pc/progress-pc.gc

```diff
@@ -1424,7 +1428,7 @@
         (((progress-screen camera-options))
          ;; camera options lines are a bit too big
          (hide-progress-icons)
-         (draw-options self 115 36 0.77)
+         (draw-options self 115 36 0.60)
```

Comment:
cool. Just reduce the line height now. It's the 36 in here. I don't remember why I set it so high.